packages/react-server-examples/bike-share/components/network-card.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import {logging, Link} from 'react-server';
+
+const logger = logging.getLogger(__LOGGER__);
+
+const NetworkCard = ({network}) => {
+	logger.info(`rendering card for network ${JSON.stringify(network)}`);
+	return (
+		<div><Link path={`/network?network=${network.id}`} frameback reuseFrame>{network.name}</Link> in {network.location.city}</div>
```

Comment:
Does this `reuseFrame` actually trigger?  I suspect it might need a truthy value.